firebase-firestore/src/test/java/com/google/firebase/firestore/util/MapperTest.java

```diff
@@ -899,7 +899,7 @@ public void setValue(String value) {
 
   private static <T> T deserialize(String jsonString, Class<T> clazz) {
     Map<String, Object> json = fromSingleQuotedString(jsonString);
-    return CustomClassMapper.convertToCustomClass(json, clazz);
+    return CustomClassMapper.convertToCustomClass(json, clazz, null);
```

Comment:
Most tests here aren't going to pass a non-null value here, so consider adding a local method that defaults the parameter to null to reduce the boilerplate of each invocation.

Reply:
Done.

Reply:
Did you intentionally not use the new method here? 😄

Reply:
Yes..because `serialize` below already refers to mapper..